README.md

```diff
@@ -181,6 +181,30 @@ It will produce files: `en-38.png`, `en-39.png`, `en-40.png` and `en-41.png`.
 
 **This script requires `pdftoppm` and `imagemagick` utilities.**
 
+### 🎭 Comparing two pages side by side (v2)
+
+Another version, which works faster, but uses a different workflow:
+
+First, switch to the `main` branch, build the `main_en.pdf` and run:
+
+```bash
+tools/compare_pages_2.sh en 38-41 before
+```
+
+If you have only 1 page to show, just type the single number, without `-`. Or use a comma-separated list of numbers.
+
+The script will save the image of the specified pages to /tmp/. Now switch to the branch you're working on, do some changes, don't forget to rebuild the .pdf, and run:
+
+```bash
+tools/compare_pages_2.sh en 38-41 after
+```
+
+The script will use the previously written "before" files as the left side, and produce the single image with all the pages combined side by side, named /tmp/en-both-38-41.png
```

Comment:
Can you make it so that it doesn't leave leftovers? I don't like the idea of having to remember to clean up after the script manually.

Reply:
These "leftovers" is the whole point of the optimization here, to avoid extract the same image again and again.

But if these leftovers are in somewhere like /tmp/homm3bgrecompare/, I don't think it's a big deal.

But feel free to reject this PR, I can just keep using it myself without sharing.

Reply:
OTOH, since it's more than twice faster than the other script, probably it would make sense to rewrite that one to use qpdf in similar way, maybe, even if extracting from both pdfs, and removing leftovers

Reply:
Will it produce the same image quailty?

Reply:
Not sure. But I recall you were saying the purpose of the images was to ensure we don't overflow pages. The produced quality is enough for that

Reply:
If I have a choice of a nice picture in 4s, and a grainy picture in 2s, I prefer the former :P

Reply:
I passed the -density param to montage to make it the same quality as the other script